include/z64actor.h

```diff
@@ -288,8 +288,17 @@ typedef struct EnAObj {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnAObjActionFunc actionFunc;
     /* 0x148 */ ColliderCylinder collision;
-    /* 0x194 */ UNK_TYPE1 pad194[0x14];
-} EnAObj; // size = 0x1A8
+} EnAObj; // size = 0x194
```

Comment:
... That was still matching?

Reply:
The only place where the size of an actor's struct is usually used is in its InitVars, but since we haven't imported the InitVars yet then we were using the hardcodded sizes from the asm, so this struct could have any size and still match.
Importing the data required the sizes to be correct so the ROM can match, then I had to correct them